source/_components/withings.markdown

```diff
@@ -0,0 +1,209 @@
+
+### {% linkable_title Account %}
+
+You must have a developer account to distribute the data. [Create a free development account](https://account.withings.com/partner/add_oauth2). 
+
+Values for your account:
+
+- Logo: Any reasonable picture will do.
+- Description: Personal app for collecting my data.
+- Contact Email: \<Your email address>
+- Callback Uri: https://your-domain-name/api/withings/callback
+- Company: Home Assistant
+
+Note: Upon saving these settings, Withings will check if your callback URI resolves. If your service isn't available, the save will fail. After you developer account is created, you can make your home assistant no longer publicly accessible (if you so choose).
+
+Once saved, the "Client Id" and "Consumer Secret" fields will be populated. Take note, you will need these later.
+
+### {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+withings:
+    client_id: <client id from previous step>
+    client_secret: <consumer secret from previous step>
+    profiles:
+        - <name of user profile>
+```
+
+Withings supports multiple profiles per account. Each profile has a person's name to help distinguish who's data you're looking at. While the profile provided here can be arbitrary, it is recommended you use the same name from the Withings profile. This will make it easier to distinguish who's data you're looking at.
+
+### {% linkable_title Authorization %}
+
+On the home assistant web page
+- Goto `Configuraton` > `General`
+  - Click `Check Config` and ensure your config looks good.
+  - After config is correct, click `Restart`.
+- After Home assistant restarts.
+  - Got the integrations page.
+  - Toward the bottom of the page, you will see list item for Withings, click it.
+  - Select the profile you intend to pull data.
+  - Click the link to be taken to the withing site to authorize the request.
+  - Withings will provide you with a list of profiles to choose from (if you have more than one). Choose the profile
+  you selected earlier. Otherwise homeassistant entity names will not match the data they are showing.
+- After you authorize at Withings, the Withings website will redirect your browser back to your callback uri.
+If your home assistant instance is not accessible publicly, you can change the url in the browser to match the 
+url of your server.
+
+That should do it. Data will begin synchronizing in 5 minutes and update every 5 minutes.
+
+## {% linkable_title FAQ %}
+
+### How can I get data for more than one profile?
+
+Create another sensor entry with the same client id and secret. Just be sure to change the profile to the name of the person you want to track. Then restart home assistant and authorize Withings for that profile.
+
+## {% linkable_title Advanced Configuration %}
+
+{% configuration %}
```

Comment:
Think that we have to move this part to under the `configuration` example.

Reply:
The document is setup in hierarchically using headers to document the order in which specific parts of the setup should occur.
```
Setup
  Account
  Configuration
  Authorization
```
Adding the advanced configuration options into this flow throws off the order.

Reply:
The configuration block serves as an overview. It would not prohibit one for going into more depth.